### PR TITLE
Jumbo frames

### PR DESCRIPTION
**Pull request by kevinpm**
Changes added to /opt/cni/bin/labmon.py to support jumbo frames (just setting the mtu of the phy and veth interfaces to 9000)

### CODE REVIEW

File: bin/labmon_cni.py [hunk 1]
```diff
@@ -170,6 +170,8 @@ def bringup(self):
         OSexec.exec(f"ip netns add dummy1")
         OSexec.exec(f"ln -sfT {netns} /var/run/netns/{containerid}")
         OSexec.exec(f"ip link add {ifname} type veth peer name {host_if_name}", retry=100)
+        OSexec.exec(f"ip link set dev {host_if_name} mtu 9000")
```
Comment (fsedano) [inline bin/labmon_cni.py]: Better to pass the MTU as a parameter from the deployment (as a label)

**Review by fsedano — CHANGES_REQUESTED** [review]

**Comment by kevinpm** [comment]
Added the MTU as a label. Check if the label exists. If yes, assign the MTU value. If not, assign the default value of 1500

**Comment by kevinpm** [comment]
Added UTs with annotations including the MTU

File: Dockerfile [hunk 2]
```diff
@@ -4,5 +4,6 @@ WORKDIR /app
 ADD bin/requirements.txt /app
 RUN pip3 install -r requirements.txt
 COPY bin/labmon_cni.py /app
+COPY bin/ut_cni.py /app
```
Comment (fsedano) [inline Dockerfile:7]: I think this is already on the master branch, can you please check?

File: bin/ut_cni.py [hunk 3]
```diff
@@ -6,41 +6,49 @@
 import logging
 
 MOCKED_K8S_PARAMS = {
-    'CNI_COMMAND' : 'ADD',
```
Comment (fsedano) [inline bin/ut_cni.py:9]: Same comment - I think it's already part of the master.

**Comment by kevinpm** [comment]
Closing

**Pull request closed without merging**